packages/broker/test/unit/plugins/storage/dataQueryEndpoint.test.ts

```diff
@@ -8,24 +8,28 @@ import {
 } from '../../../../src/plugins/storage/dataQueryEndpoint'
 import { toObject } from '../../../../src/plugins/storage/DataQueryFormat'
 import { Storage } from '../../../../src/plugins/storage/Storage'
-import { PassThrough } from 'stream'
+import { PassThrough, Readable } from 'stream'
 import { ContentType, MessageID, StreamMessage, toStreamID } from '@streamr/protocol'
-import { MetricsContext, toEthereumAddress, hexToBinary, utf8ToBinary } from '@streamr/utils'
+import { MetricsContext, toEthereumAddress, hexToBinary, utf8ToBinary, toLengthPrefixedFrame } from '@streamr/utils'
+import { convertStreamMessageToBytes } from '@streamr/trackerless-network'
 
 const createEmptyStream = () => {
     const stream = new PassThrough()
     stream.push(null)
     return stream
 }
 
+const intoBinaryStream = (msg: StreamMessage[]): Readable => {
```

Comment:
```suggestion
const createOutputStream = (msg: StreamMessage[]): Readable => {
```

Also could remove `createEmptyStream` and use `createOutputStream([])` instead of it?